docs/CONTRIBUTING.md

```diff
@@ -41,44 +42,52 @@ Be sure to mention that you're using Connectors and also let us know what servic
 You can also find us in the `#enterprise-search` channel of the [Elastic Community Slack](http://elasticstack.slack.com).
 
 ## Types of contributions
-### Initial contribution
+
+### Customize a connector
```

Comment:
My 2 cents, we will never merge a "contribution" that is a connector customization. If it's a generic improvement, I'd think that falls under "enhancement." If it's specific to the customer's own use case, they're welcome to make changes in their fork, but we will not consider customizations for contributing back. WDYT?

Reply:
I think we can remove the the category of "Types of contributions" entirely and just have separate sections for "Add new connector" and "Customize a connector". We can incorporate your point about not merging customizations as preamble to the customization option, along with any support-relevant info.

Reply:
> Right, what's the deal with Elastic supporting customized connectors?

I think in that case we support the _framework_ but no longer the connector itself? CC @danajuratoni @ppf2 

> I think we can remove the the category of "Types of contributions" entirely and just have separate sections for "Add new connector" and "Customize a connector". 

I'm fine with that, but at that point it might not make sense for this to be a `CONTRIBUTING.md`. Usually I see these files used _only_ to express to folks what the expectations are if you're going to file issues or PRs. Personally, I'd put stuff for customizing connectors under DEVELOPING.md, or something else. But I think we have a lot of options here, and my opinions aren't super strongly held.

Reply:
No worries. You know far better than me where to draw the line between contributing guidelines and developing guidelines, we can easily move the customization stuff to DEVELOPING.md

Reply:
I agree with Sean that this section sounds like: "Adapt existing connector to my specific needs". In that case we shouldn't treat is as contribution and the point about required unit tests % is not necessary. I would also move it away from CONTRIBUTING, Developing sounds ok!

Reply:
I view customized connector clients, converted native connectors (and then customized), and custom connectors in the same category - they are not supported by Elastic.  Users will have to first demonstrate that the issue is occurring in a non-customized connector.   If the user believes that there is a bug in the connector framework, we will need an isolated test case demonstrating that the issue is in the framework, not the customized connector.